service/service_construction.go

```diff
@@ -309,21 +311,28 @@ func (s ConstructionService) ConstructionParse(
 	if err != nil {
 		return nil, wrapError(errInternalError, err)
 	}
-	var ops []*types.Operation
-	var checkFrom *string
-	var wrappedErr *types.Error
 
+	var (
+		ops        []*types.Operation
+		checkFrom  *string
+		wrappedErr *types.Error
+	)
 	if len(tx.Data) != 0 {
 		unwrapMethodID := getMethodID(unwrapFnSignature)
+		transferMethodID := getMethodID(transferFnSignature)
 		if hexutil.Encode(tx.Data[:4]) == hexutil.Encode(unwrapMethodID) {
```

Comment:
was gonna put a comment about the switch statement but looks like the linter caught it first haha

Reply:
af10ff8